internals/generators/container/slice.test.js.hbs

```diff
@@ -0,0 +1,28 @@
+import { reducer /* defaultAction */ } from '../slice';
+
+describe('{{ camelCase name }}Slice actions and reducer', () => {
+  let state;
+
+  beforeEach(() => {
+    state = {
+      // default state params here
+    };
+  });
+
+  it('returns the initial state', () => {
+    const expectedResult = state;
+    expect(reducer(undefined, {})).toEqual(expectedResult);
+  });
+
+  /**
```

Comment:
minor thing but I don't think you can uncomment multi-line comments with `CMD + /`

Reply:
Fixed.